### PR TITLE
[RHACS] Updates to Release Notes for 3.70.2 Patch

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):

Merge to: `rhacs-docs-3.70`
Cherry pick to: `rhacs-docs`

[Issue](https://issues.redhat.com/browse/ROX-12852)

[Link to docs preview](https://openshift-docs-k3s6dpz0l-kcarmichael08.vercel.app/openshift-acs/master/release_notes/370-release-notes.html)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by 0x656b694d** [comment]
LGTM

File: release_notes/370-release-notes.adoc [hunk 1]
```diff
@@ -168,6 +169,17 @@ In the table, features are marked with the following statuses:
 [id="bug-fixes-3670"]
 == Bug fixes
 
+[id="fixed-in-version-3702"]
+=== Resolved in version 3.70.2
+
+Release date: 5 October 2022:
```
Comment (bergerhoffer) [inline release_notes/370-release-notes.adoc:175]: There is an extra colon here, would you want to fix it before I merge, or after?

**Review by bergerhoffer — COMMENTED** [review]
LGTM for merging

**Comment by bergerhoffer** [comment]
@kcarmichael08 The CP to rhacs-docs failed. Can you please submit a manual CP for it?

**Pull request merged**